### DISCUSSION TITLE
Compact Header Guide for Limited Vertical Space

### DISCUSSION THREAD

**Discussion by reichst1SICKAG** [synergy-design-system/synergy-design-system#534] [General]
In certain Apps, vertical space is very limited, and a more compact header with reduced margin can improve usability and design.

**How to Create a Compact Header:**
To adjust the header for better space management, follow these steps:

1. Identify the Current Header:
- Ensure you are working with the appropriate syn-header component.
- Confirm the current margin is set to spacing/large (24px).
<img width="782" alt="header-spacing-large" src="https://github.com/user-attachments/assets/0d3bcad1-a28c-4e48-b607-c99a9e03ff9c">

2. Reduce the Header Margin:
- Modify the spacing token of the margin from spacing/large to spacing/medium.
- Old Margin: 24px
- New Margin: 16px
<img width="793" alt="header-spacing-medium" src="https://github.com/user-attachments/assets/881cf6b5-e09b-4ae5-b828-609baa8da9ee">

This adjustment will make the header more compact and better suited for apps with limited vertical space.

**Important Notes:**
- This is not a Synergy Component or Template:
   - Please note that this is a how-to guide and is not part of any predefined Synergy component.
   - This solution is for custom applications where space constraints are an issue.

- Avoid Using Prio-Nav with Compact Header:
    - The syn-prio-nav (priority navigation) does not visually align well with the compact header due to spacing constraints.
    - Recommendation: If you implement the compact header, avoid pairing it with prio-nav to maintain visual consistency.

**Comment by schilchSICKAG** [comment]
Just to chime in, we also got another request for this recently. @klaarseSICKAG knows more details here.

**Comment by reichst1SICKAG** [comment]
New compact header is now default, see [syn-header](https://synergy-design-system.github.io/?path=/docs/components-syn-header--docs)
